site/docs/trino-prestodb.md

```diff
@@ -17,4 +17,11 @@
 
 # Trino (formerly Presto SQL)
 
-The Iceberg connector is available with Trino. It is feature complete and usage details can be found in the [documentation](https://trino.io/docs/current/connector/iceberg.html)
+The Iceberg connector is available with PrestoSQL, which is rebranded as Trino since January 2021.
```

Comment:
The title already states that Trino used to be called Presto SQL, stating it again in the paragraph seems redundant. Please consider replacing this with: "The documentation for Iceberg's Trino connector has moved to the Trino website. Please consult the [Iceberg section](https://trino.io/docs/current/connector/iceberg.html) of the Trino docs for more information."